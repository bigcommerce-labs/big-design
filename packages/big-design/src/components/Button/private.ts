import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
import styled from 'styled-components';

import { ButtonProps, StyleableButton as _StyleableButton } from './Button';

export const StyleableButton = _StyleableButton;

export const MessagingButton = styled(_StyleableButton)<ButtonProps>`
  background-color: transparent;
  border: ${({ theme }) => theme.border.none};
  color: ${({ theme }) => theme.colors.secondary70};
  height: auto;
  padding: ${({ theme }) => theme.spacing.none};

  &:active {
    background-color: ${({ theme }) => theme.colors.primary20};
  }

  &:focus {
    box-shadow: ${({ theme }) => `0 0 0 ${theme.spacing.xxSmall} ${theme.colors.primary20}`};
  }

  &:hover:not(:active) {
    background-color: ${({ theme }) => theme.colors.primary10};
  }

  &[disabled] {
    color: ${({ theme }) => theme.colors.secondary30};
  }
`;

MessagingButton.defaultProps = { theme: defaultTheme };
