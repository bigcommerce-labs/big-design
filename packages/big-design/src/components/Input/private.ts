export { StyledInput, StyledInputWrapper, type StyledInputWrapperProps } from './styled';
