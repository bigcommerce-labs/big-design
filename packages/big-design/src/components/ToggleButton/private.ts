import { ToggleButtonProps as _ToggleButtonProps } from './ToggleButton';
export { ToggleButton } from './ToggleButton';
export type ToggleButtonProps = _ToggleButtonProps;
