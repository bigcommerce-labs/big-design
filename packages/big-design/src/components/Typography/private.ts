export {
  StyleableText,
  StyleableSmall,
  StyleableH0,
  StyleableH1,
  StyleableH2,
  StyleableH3,
  StyleableH4,
} from './Typography';
