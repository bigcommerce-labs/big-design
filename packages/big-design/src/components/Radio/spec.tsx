import { fireEvent, render } from '@testing-library/react';
import 'jest-styled-components';
import React from 'react';

import { Radio } from './index';
import { StyleableRadio } from './private';

test('render Radio (checked)', () => {
  const { container } = render(<Radio label="Checked" name="test-group" checked={true} onChange={() => null} />);

  expect(container.firstChild).toMatchSnapshot();
});

test('render Radio (unchecked)', () => {
  const { container } = render(<Radio label="Unchecked" name="test-group" checked={false} onChange={() => null} />);

  expect(container.firstChild).toMatchSnapshot();
});

test('render Radio (checked + disabled)', () => {
  const { container } = render(
    <Radio label="Checked" name="test-group" checked={true} onChange={() => null} disabled />,
  );

  expect(container.firstChild).toMatchSnapshot();
});

test('render Radio (unchecked + disabled)', () => {
  const { container } = render(
    <Radio label="Unchecked" name="test-group" checked={false} onChange={() => null} disabled />,
  );

  expect(container.firstChild).toMatchSnapshot();
});

test('has correct value when checked', () => {
  const { getByTestId } = render(<Radio label="Checked" checked={true} onChange={() => null} data-testid="radio" />);
  const radio = getByTestId('radio') as HTMLInputElement;

  expect(radio.checked).toBe(true);
});

test('has correct value when unchecked', () => {
  const { getByTestId } = render(<Radio label="Unchecked" checked={false} onChange={() => null} data-testid="radio" />);
  const radio = getByTestId('radio') as HTMLInputElement;

  expect(radio.checked).toBe(false);
});

test('triggers onChange when clicking the radio button', () => {
  const onChange = jest.fn();
  const { getByTestId } = render(<Radio label="Checked" checked={false} onChange={onChange} data-testid="radio" />);
  const radio = getByTestId('radio') as HTMLInputElement;

  fireEvent.click(radio);

  expect(onChange).toHaveBeenCalled();
});

test('triggers onChange when clicking styled and text label', () => {
  const onChange = jest.fn();
  const { container } = render(<Radio label="Checked" checked={false} onChange={onChange} />);

  const labels = container.querySelectorAll('label');

  labels.forEach(label => fireEvent.click(label));

  expect(onChange).toHaveBeenCalledTimes(2);
});

test('forwards ref', () => {
  const ref = React.createRef<HTMLInputElement>();

  const { container } = render(<Radio ref={ref} label="Checked" />);
  const input = container.querySelector('input');

  expect(input).toBe(ref.current);
});

test('does not forward styles', () => {
  const { container } = render(<Radio label="Checked" className="test" />);

  expect(container.getElementsByClassName('test').length).toBe(0);
});

test('private version forwards styles', () => {
  const { container } = render(<StyleableRadio label="Checked" className="test" />);

  expect(container.getElementsByClassName('test').length).toBe(1);
});
