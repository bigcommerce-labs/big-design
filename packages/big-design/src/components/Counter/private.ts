import { remCalc, theme as defaultTheme } from '@bigcommerce/big-design-theme';
import styled from 'styled-components';

import { StyledInput, StyledInputWrapper, StyledInputWrapperProps } from '../Input/styled';

export const StyledCounterWrapper = styled(StyledInputWrapper)<StyledInputWrapperProps>`
  padding-left: ${({ theme }) => theme.spacing.xxSmall};
  padding-right: ${({ theme }) => theme.spacing.xxSmall};
  width: ${remCalc(125)};
  max-width: ${remCalc(320)};
`;

export const StyledCounterInput = styled(StyledInput)`
  text-align: center;
  padding: ${({ theme }) => theme.spacing.none};
`;

StyledCounterInput.defaultProps = { theme: defaultTheme };
StyledCounterWrapper.defaultProps = { theme: defaultTheme };
