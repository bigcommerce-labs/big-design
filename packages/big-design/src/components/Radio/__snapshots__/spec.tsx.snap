// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Radio (checked + disabled) 1`] = `
.c4 {
  margin-left: 0.5rem;
}

.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}

.c1:focus + .c6 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c1:focus + .c7 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3 {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: border-color,box-shadow;
  transition-property: border-color,box-shadow;
  background-color: #F6F7FC;
  border: 1px solid #D9DCE9;
  border-color: #3C64F4;
  border-radius: 50%;
  color: #FFFFFF;
  cursor: pointer;
  display: inline-block;
  height: 1.25rem;
  position: relative;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1.25rem;
  cursor: not-allowed;
  border-color: #D9DCE9;
}

.c1:focus + .c3 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3:after {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: opacity;
  transition-property: opacity;
  background-color: #B4BAD1;
  border-radius: 50%;
  content: '';
  height: 0.75rem;
  left: 50%;
  opacity: 0;
  position: absolute;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  width: 0.75rem;
}

.c3:after {
  opacity: 1;
}

.c5 {
  color: #313440;
  margin: 0 0 1rem;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.5rem;
  cursor: pointer;
  cursor: not-allowed;
}

.c5:last-child {
  margin-bottom: 0;
}

<div
  class="c0"
>
  <input
    aria-labelledby="bd-radio_label-2"
    checked=""
    class="c1 c2"
    disabled=""
    id="bd-radio-1"
    name="test-group"
    type="radio"
  />
  <label
    aria-hidden="true"
    class="c3"
    disabled=""
    for="bd-radio-1"
  />
  <div
    class="c4"
  >
    <label
      aria-hidden="true"
      class="c5"
      disabled=""
      for="bd-radio-1"
      id="bd-radio_label-2"
    >
      Checked
    </label>
  </div>
</div>
`;

exports[`render Radio (checked) 1`] = `
.c4 {
  margin-left: 0.5rem;
}

.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}

.c3 {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: border-color,box-shadow;
  transition-property: border-color,box-shadow;
  background-color: #FFFFFF;
  border: 1px solid #D9DCE9;
  border-color: #3C64F4;
  border-radius: 50%;
  color: #FFFFFF;
  cursor: pointer;
  display: inline-block;
  height: 1.25rem;
  position: relative;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1.25rem;
}

.c3:hover {
  border-color: #2852EB;
}

.c1:focus + .c3 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3:after {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: opacity;
  transition-property: opacity;
  background-color: #3C64F4;
  border-radius: 50%;
  content: '';
  height: 0.75rem;
  left: 50%;
  opacity: 0;
  position: absolute;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  width: 0.75rem;
}

.c3:after {
  opacity: 1;
}

.c5 {
  color: #313440;
  margin: 0 0 1rem;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.5rem;
  cursor: pointer;
}

.c5:last-child {
  margin-bottom: 0;
}

<div
  class="c0"
>
  <input
    aria-labelledby="bd-radio_label-2"
    checked=""
    class="c1 c2"
    id="bd-radio-1"
    name="test-group"
    type="radio"
  />
  <label
    aria-hidden="true"
    class="c3"
    for="bd-radio-1"
  />
  <div
    class="c4"
  >
    <label
      class="c5"
      for="bd-radio-1"
      id="bd-radio_label-2"
    >
      Checked
    </label>
  </div>
</div>
`;

exports[`render Radio (unchecked + description object) 1`] = `
.c6 {
  color: #313440;
  margin: 0 0 1rem;
  color: #5E637A;
  font-size: 0.875rem;
  font-weight: 400;
  line-height: 1.25rem;
  margin: 0 0 0.75rem;
}

.c6:last-child {
  margin-bottom: 0;
}

.c7 {
  -webkit-transition: all 70ms ease-out;
  transition: all 70ms ease-out;
  -webkit-transition-property: color;
  transition-property: color;
  color: #3C64F4;
  cursor: pointer;
  font-size: 1rem;
  font-weight: 400;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c7:active {
  color: #0024A6;
}

.c7:hover:not(:active) {
  color: #0024A6;
}

.c8 {
  font-size: 0.875rem;
}

.c4 {
  margin-left: 0.5rem;
}

.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}

.c1:focus + .c9 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3 {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: border-color,box-shadow;
  transition-property: border-color,box-shadow;
  background-color: #FFFFFF;
  border: 1px solid #D9DCE9;
  border-color: #D9DCE9;
  border-radius: 50%;
  color: #FFFFFF;
  cursor: pointer;
  display: inline-block;
  height: 1.25rem;
  position: relative;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1.25rem;
}

.c3:hover {
  border-color: #B4BAD1;
}

.c1:focus + .c3 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3:after {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: opacity;
  transition-property: opacity;
  background-color: #3C64F4;
  border-radius: 50%;
  content: '';
  height: 0.75rem;
  left: 50%;
  opacity: 0;
  position: absolute;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  width: 0.75rem;
}

.c5 {
  color: #313440;
  margin: 0 0 1rem;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.5rem;
  cursor: pointer;
}

.c5:last-child {
  margin-bottom: 0;
}

<div
  class="c0"
>
  <input
    aria-labelledby="bd-radio_label-2"
    class="c1 c2"
    id="bd-radio-1"
    name="test-group"
    type="radio"
  />
  <label
    aria-hidden="true"
    class="c3"
    for="bd-radio-1"
  />
  <div
    class="c4"
  >
    <label
      class="c5"
      for="bd-radio-1"
      id="bd-radio_label-2"
    >
      Unchecked
    </label>
    <p
      class="c6"
    >
      description
       
      <a
        class="c7 c8"
        href="bar"
        target="foo"
      >
        learn more
      </a>
    </p>
  </div>
</div>
`;

exports[`render Radio (unchecked + description text) 1`] = `
.c6 {
  color: #313440;
  margin: 0 0 1rem;
  color: #5E637A;
  font-size: 0.875rem;
  font-weight: 400;
  line-height: 1.25rem;
  margin: 0 0 0.75rem;
}

.c6:last-child {
  margin-bottom: 0;
}

.c4 {
  margin-left: 0.5rem;
}

.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}

.c1:focus + .c7 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3 {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: border-color,box-shadow;
  transition-property: border-color,box-shadow;
  background-color: #FFFFFF;
  border: 1px solid #D9DCE9;
  border-color: #D9DCE9;
  border-radius: 50%;
  color: #FFFFFF;
  cursor: pointer;
  display: inline-block;
  height: 1.25rem;
  position: relative;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1.25rem;
}

.c3:hover {
  border-color: #B4BAD1;
}

.c1:focus + .c3 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3:after {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: opacity;
  transition-property: opacity;
  background-color: #3C64F4;
  border-radius: 50%;
  content: '';
  height: 0.75rem;
  left: 50%;
  opacity: 0;
  position: absolute;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  width: 0.75rem;
}

.c5 {
  color: #313440;
  margin: 0 0 1rem;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.5rem;
  cursor: pointer;
}

.c5:last-child {
  margin-bottom: 0;
}

<div
  class="c0"
>
  <input
    aria-labelledby="bd-radio_label-2"
    class="c1 c2"
    id="bd-radio-1"
    name="test-group"
    type="radio"
  />
  <label
    aria-hidden="true"
    class="c3"
    for="bd-radio-1"
  />
  <div
    class="c4"
  >
    <label
      class="c5"
      for="bd-radio-1"
      id="bd-radio_label-2"
    >
      Unchecked
    </label>
    <p
      class="c6"
    >
      description
    </p>
  </div>
</div>
`;

exports[`render Radio (unchecked + disabled) 1`] = `
.c4 {
  margin-left: 0.5rem;
}

.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}

.c1:focus + .c6 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c1:focus + .c7 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c1:focus + .c8 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3 {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: border-color,box-shadow;
  transition-property: border-color,box-shadow;
  background-color: #F6F7FC;
  border: 1px solid #D9DCE9;
  border-color: #D9DCE9;
  border-radius: 50%;
  color: #FFFFFF;
  cursor: pointer;
  display: inline-block;
  height: 1.25rem;
  position: relative;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1.25rem;
  cursor: not-allowed;
  border-color: #D9DCE9;
}

.c1:focus + .c3 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3:after {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: opacity;
  transition-property: opacity;
  background-color: #B4BAD1;
  border-radius: 50%;
  content: '';
  height: 0.75rem;
  left: 50%;
  opacity: 0;
  position: absolute;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  width: 0.75rem;
}

.c5 {
  color: #313440;
  margin: 0 0 1rem;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.5rem;
  cursor: pointer;
  cursor: not-allowed;
}

.c5:last-child {
  margin-bottom: 0;
}

<div
  class="c0"
>
  <input
    aria-labelledby="bd-radio_label-2"
    class="c1 c2"
    disabled=""
    id="bd-radio-1"
    name="test-group"
    type="radio"
  />
  <label
    aria-hidden="true"
    class="c3"
    disabled=""
    for="bd-radio-1"
  />
  <div
    class="c4"
  >
    <label
      aria-hidden="true"
      class="c5"
      disabled=""
      for="bd-radio-1"
      id="bd-radio_label-2"
    >
      Unchecked
    </label>
  </div>
</div>
`;

exports[`render Radio (unchecked) 1`] = `
.c4 {
  margin-left: 0.5rem;
}

.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}

.c1:focus + .c6 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3 {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: border-color,box-shadow;
  transition-property: border-color,box-shadow;
  background-color: #FFFFFF;
  border: 1px solid #D9DCE9;
  border-color: #D9DCE9;
  border-radius: 50%;
  color: #FFFFFF;
  cursor: pointer;
  display: inline-block;
  height: 1.25rem;
  position: relative;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1.25rem;
}

.c3:hover {
  border-color: #B4BAD1;
}

.c1:focus + .c3 {
  box-shadow: 0 0 0 0.25rem #DBE3FE;
}

.c3:after {
  -webkit-transition: all 150ms ease-out;
  transition: all 150ms ease-out;
  -webkit-transition-property: opacity;
  transition-property: opacity;
  background-color: #3C64F4;
  border-radius: 50%;
  content: '';
  height: 0.75rem;
  left: 50%;
  opacity: 0;
  position: absolute;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  width: 0.75rem;
}

.c5 {
  color: #313440;
  margin: 0 0 1rem;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.5rem;
  cursor: pointer;
}

.c5:last-child {
  margin-bottom: 0;
}

<div
  class="c0"
>
  <input
    aria-labelledby="bd-radio_label-2"
    class="c1 c2"
    id="bd-radio-1"
    name="test-group"
    type="radio"
  />
  <label
    aria-hidden="true"
    class="c3"
    for="bd-radio-1"
  />
  <div
    class="c4"
  >
    <label
      class="c5"
      for="bd-radio-1"
      id="bd-radio_label-2"
    >
      Unchecked
    </label>
  </div>
</div>
`;
